src/app/utils/safeHtml.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({ name: 'safeHtml'})
+export class SafeHtmlPipe implements PipeTransform  {
+  constructor(private sanitized: DomSanitizer) {}
+  transform(value) {
+    return this.sanitized.bypassSecurityTrustHtml(value);
```

Comment:
Why bypass the sanitization and not just sanitize it using the following code?
```suggestion
    return this.sanitized.sanitize(1, value);
```
I understand we already do some sanitization on the backend but does it hurt to do some on the frontend as well?

Reply:
I tested it and I thought it messed up the styling of the `build status icons` but apparently that was caused by something else.